### DISCUSSION TITLE
Skip "press enter to return to lazygit"

### DISCUSSION THREAD

**Discussion by theduke** [jesseduffield/lazygit#1462] [Q&A]
Is there a way to skip the "press enter to return to lazygit" step?

I mostly use the editor for commits, and that extra step is quite annoying.

**Comment by jesseduffield** [comment]
This happens to be implemented in lazydocker, but not lazygit. It should be easy to implement. Could you raise an issue for this, if one doesn't already exist?

**Comment by joshmedeski** [comment]
This has started working for me properly on the latest version 👍

**Comment by Tej12345** [comment]
FOUND THE ANSWER!!!
Add to config.yml:
```
promptToReturnFromSubprocess: false
```
Source: https://github.com/jesseduffield/lazygit/issues/1576#issuecomment-1111690534
